### PR TITLE
edencommon fb303 fbthrift fizz folly wangle watchman 2023.06.26.00

### PR DESCRIPTION
**Pull request by razvanazamfirei**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/edencommon.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Edencommon < Formula
   desc "Shared library for Watchman and Eden projects"
   homepage "https://github.com/facebookexperimental/edencommon"
-  url "https://github.com/facebookexperimental/edencommon/archive/refs/tags/v2023.06.12.00.tar.gz"
-  sha256 "8cfd53508ba31a5a830987a7c99ecf2dc25541a76f82e836c260e6c53e4d793b"
+  url "https://github.com/facebookexperimental/edencommon/archive/refs/tags/v2023.06.26.00.tar.gz"
+  sha256 "0b6987a55c7c57a2b22fa40ffcbcafe4421a0fb9bb1bc0b38d6f7df731f1921f"
   license "MIT"
   revision 1
```
Comment (carlocab) [inline Formula/edencommon.rb:7]:
```suggestion
```
Same below.

**Comment by carlocab** [comment]
folly needs https://github.com/facebook/folly/commit/e74fe5c99dd9d96b80b17f8396ce2568f2becb45.

**Comment by carlocab** [comment]
`fizz` test failure:
```
  Undefined symbols for architecture arm64:
    "fizz::Sha256::BlankHash", referenced from:
        std::__1::unique_ptr<fizz::KeyDerivationImpl, std::__1::default_delete<fizz::KeyDerivationImpl> > fizz::KeyDerivationImpl::make<fizz::Sha256>(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in test-e4dafb.o
        fizz::HandshakeContextImpl<fizz::Sha256>::getBlankContext() const in test-e4dafb.o
        fizz::KeyDerivationImpl fizz::KeyDerivationImpl::create<fizz::Sha256>(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in test-e4dafb.o
    "fizz::Sha384::BlankHash", referenced from:
        std::__1::unique_ptr<fizz::KeyDerivationImpl, std::__1::default_delete<fizz::KeyDerivationImpl> > fizz::KeyDerivationImpl::make<fizz::Sha384>(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in test-e4dafb.o
        fizz::HandshakeContextImpl<fizz::Sha384>::getBlankContext() const in test-e4dafb.o
        fizz::KeyDerivationImpl fizz::KeyDerivationImpl::create<fizz::Sha384>(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) in test-e4dafb.o
```

**Comment by carlocab** [comment]
`fizz::Sha256::BlankHash` seems to have gone missing in this version.

**Review by carlocab — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**